src/Symfony/Bundle/FrameworkBundle/EventListener/SessionListener.php

```diff
@@ -14,10 +14,14 @@
 use Symfony\Component\HttpKernel\EventListener\SessionListener as BaseSessionListener;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 
+@trigger_error(sprintf('The %s class is deprecated since version 3.3 and will be removed in 4.0. Use %s instead.', SessionListener::class, BaseSessionListener::class), E_USER_DEPRECATED);
```

Comment:
Not sure if we can deprecate it. What if we decide to remove the getter feature in 3.4? We would have to underprecate this class.